### DISCUSSION TITLE
OpenClash Rpi3B

### DISCUSSION THREAD

**Discussion by piterleopard** [helmiau/HelmiWrt-OS#40] [Q&A]
Izin nanya bg Helmi, kenapa openclash di raspberry mode redirhost(tun mode) bengong ya, tapi kalau mode redirhost aja bisa. Masalahnya kalau mode redirhost aja gak support game dan vc, sedangkan yang fw untuk STB nya bisa redirhost tun mode (support game dan VC). Ada solusi kah bg biar openclash di rpi3 support vc dan game? Terimakasih.

**Comment by helmiau** [comment]
Mungkin bisa dicoba pke mode yg lain bang

**Comment by malikshi** [comment]
Build/compiled hari ini harusnya sudah fix. Yang sebelumnya kemungkinan error karena binary clash, clash tun, clash game nya tidak ada
